test/e2e/subscription_e2e_test.go

```diff
@@ -1253,6 +1253,141 @@ var _ = Describe("Subscription", func() {
 		require.NoError(GinkgoT(), err)
 		require.Len(GinkgoT(), installPlan.Status.CatalogSources, 1)
 	})
+
+	It("creation with multiple dependencies", func() {
+
+		defer cleaner.NotifyTestComplete(true)
+
+		kubeClient := newKubeClient()
+		crClient := newCRClient()
+
+		permissions := deploymentPermissions()
+
+		crdPlural := genName("ins")
+		crdName := crdPlural + ".cluster.com"
+		crdPlural2 := genName("ins")
+		crdName2 := crdPlural2 + ".cluster.com"
+
+		crd := apiextensions.CustomResourceDefinition{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: crdName,
+			},
+			Spec: apiextensions.CustomResourceDefinitionSpec{
+				Group:   "cluster.com",
+				Version: "v1alpha1",
+				Names: apiextensions.CustomResourceDefinitionNames{
+					Plural:   crdPlural,
+					Singular: crdPlural,
+					Kind:     crdPlural,
+					ListKind: "list" + crdPlural,
+				},
+				Scope: "Namespaced",
+			},
+		}
+
+		crd2 := apiextensions.CustomResourceDefinition{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: crdName2,
+			},
+			Spec: apiextensions.CustomResourceDefinitionSpec{
+				Group:   "cluster.com",
+				Version: "v1alpha1",
+				Names: apiextensions.CustomResourceDefinitionNames{
+					Plural:   crdPlural2,
+					Singular: crdPlural2,
+					Kind:     crdPlural2,
+					ListKind: "list" + crdPlural2,
+				},
+				Scope: "Namespaced",
+			},
+		}
+
+		// Create CSV
```

Comment:
This test could use a few more comments and/or some better naming. I wrote this out to make sure I was following:

CatSrc:
 - Package A
    - Default Channel: Stable
    - Channel Stable:
       -  Operator A (Requires: CRD 1, CRD 2 )
    - Channel Alpha:
       -  Operator ABC (Provides: CRD 1, CRD 2 )
- Package B
    - Default Channel: Stable
    - Channel Stable: 
       - Operator B (Provides: CRD)

CatSrc2:
 - Package B
    - Default Channel: Stable
    - Channel Stable: 
       - Operator C (Provides: CRD 2)

Then create a subscription:

CatalogSource: CatSrc      
Package: Package A,
Channel:  Stable,
StartingCSV:   CSV A

Check installed:

CSV A, CSV B, CSV C

CSV A required B and C but didn't get them from Package A ✅